bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -148,13 +177,65 @@ void ReplicaImp::messageHandler(MessageBase *msg) {
       }
     }
   }
-  if (!isCollectingState() && validateMessage(trueTypeObj)) {
+
+  if (!isCollectingState()) {
+    // If replica is not doing state transfer, then for messages whose validation is needed
```

Comment:
Please re-phrase the comment. Not clear.

Reply:
Thanks @yuliasherman, let me rephrase.